src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -791,7 +791,12 @@
 				Value="{ThemeResource SystemControlForegroundBaseHighBrush}" />
 		<Setter Property="UseSystemFocusVisuals"
 				Value="True" />
-		<Setter Property="Template">
+		<Setter Property="FontFamily"
```

Comment:
Did you pick this from the UWP `generic.xaml` file ?

Reply:
Yes, it is copied from ComboBox template, from DefaultComboBoxStyle, lines 514-517.
I think it was only omission (i.e., setting fontsize & fontfamily was missing in ComboBoxItem).